lib/generate-svg.js

```diff
@@ -87,7 +101,7 @@ module.exports = (files = [], options = {}) => {
                 sprite = addVarNamespace(sprite);
             }
 
-            SVGOptimizer.optimize(sprite).then((output) => resolve({
+            SVGOptimizer.optimize(sprite, {path: file}).then((output) => resolve({
```

Comment:
Is there documentation on these options? I can't seem to find them 🤔

Reply:
Nope. There is not even prefixIds doc. You can check the plugin source https://github.com/svg/svgo/blob/master/plugins/prefixIds.js (l130)

Reply:
Nope, no doc. I just check the prefixId plugin code https://github.com/svg/svgo/blob/master/plugins/prefixIds.js (l130)

Reply:
I was afraid so, looks like the plugin has been available for over a year a now so we should be good there. The plugin is disabled by default though, do you manually enable it? https://github.com/svg/svgo/blob/ee065be5a88c85b976db601c13eb6e55650a7d68/plugins/prefixIds.js#L5

The plugin seems to be aimed at all selectors (and usages of these selectors) as described in the [original PR](https://github.com/svg/svgo/pull/700) so it should be separate from the prefix this plugin is adding. Could you explain why this is needed in some more detail?